src/juxt.js

```diff
@@ -0,0 +1,25 @@
+var __ = require('./__');
+var _curry1 = require('./internal/_curry1');
+var apply = require('./apply');
+var map = require('./map');
+
+/**
+ * juxt applies a list of functions to a list of values.
+ *
+ * @func
+ * @memberOf R
+ * @since v0.18.1
```

Comment:
Please remove this line. We'll include the `@since` annotation when we include this file in a release.